docs/native-modules-nuget.md

```diff
@@ -0,0 +1,86 @@
+---
+id: native-modules-nuget
+title: Using react-native-windows NuGet packages
+---
+
+>**This documentation and the underlying platform code is a work in progress.**
+
+The default for react-native-windows has been to build all code from source. This includes building all the code shipped by the react-native-windows team in the npm package from source. This code can be quite large and take a long time to build as well as require a performent computer. Some configurations have problems building this code with only 8Gb of memory. Especially if you are used to working only with managed code this can be a big surprise.
```

Comment:
```suggestion
The default for react-native-windows has been to build all code from source. This includes building all the code shipped by the react-native-windows team in the npm package from source. This code can be quite large and take a long time to build as well as require a performant computer. Some configurations have problems building this code with only 8 GB of memory. Especially if you are used to working only with managed code, this can be a big surprise.
```